plugins/auth-backend-module-google-provider/README.md

```diff
@@ -1,5 +1,8 @@
-# Auth Backend Module - Google Provider
+# Auth Module: Google Provider
+
+This module provides an Google auth provider implementation for `@backstage/plugin-auth-backend`.
```

Comment:
```suggestion
This module provides a Google auth provider implementation for `@backstage/plugin-auth-backend`.
```

also for the last part ... is the package name the core of what we want to convey? 🤔 I wonder what the most useful oneliner here would be. `...implementation for the [auth backend plugin](url to repo)` perhaps?

Reply:
Yeah not sure tbh, feeling this is good enough for now 😅